man/systemd.exec.xml

```diff
@@ -1655,6 +1658,18 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>RuntimeDirectoryPreserve=</varname></term>
+
+        <listitem><para>Takes one of <option>never</option>, <option>restart</option>, or <option>always</option>.
+        If set to <option>no</option> (the default), the directories specified in <varname>RuntimeDirectory=</varname>
+        are always removed when the service stops. If set to <option>restart</option> the directories are preserved
+        when the service is both automatically and manually restarted. Here, the automatic restart means the operation
+        specified in <varname>Restart=</varname>, and manual restart means the one triggered by
+        <command>systemctl restart foo.service</command>.
+        </para></listitem>
```

Comment:
IIUC always means the directory is never removed, except on reboot (it's a tmpfs after all)

it might be worth saying that explicitely...